sklearn/model_selection/_search.py

```diff
@@ -643,6 +655,23 @@ def _run_search(self, evaluate_candidates):
         """
         raise NotImplementedError("_run_search not implemented.")
 
+    def _check_refit_for_multimetric(self, scores_dict):
+        """Check score contains the string in refit"""
+        multimetric_refit_msg = ("For multi-metric scoring, the parameter "
+                                 "refit must be set to a scorer key or a "
+                                 "callable to refit an estimator with the "
+                                 "best parameter setting on the whole "
+                                 "data and make the best_* attributes "
+                                 "available for that metric. If this is "
+                                 "not needed, refit should be set to "
+                                 "False explicitly. %r was passed."
+                                 % self.refit)
+        if self.refit is not False and (
+                not isinstance(self.refit, str) or
```

Comment:
we could almost break the condition into variable with meaningful name. It will be easier to read the if statement